src/components/NavBar/index.tsx

```diff
@@ -1,18 +1,67 @@
-import Styled from "@emotion/styled";
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
 import { palette } from "@leafygreen-ui/palette";
+import Icon from "components/Icon";
+import PopoverButton from "components/PopoverButton";
+import SearchBar from "components/SearchBar";
+import { StyledLink, StyledRouterLink } from "components/styles";
 import { size } from "constants/tokens";
 
 const { gray, white } = palette;
 
-const NavBar: React.FC = () => <Container>OUR TRUSTY NAVBAR</Container>;
+const NavBar: React.FC = () => (
+  <Container>
+    <FlexContainer>
+      <LinkContainer>
+        <Icon glyph="LobsterLogo" />
+        <StyledLink css={navLinkStyles} href="https://wiki.corp.mongodb.com">
+          Wiki
+        </StyledLink>
+        <StyledRouterLink css={navLinkStyles} to="/upload">
+          Upload
+        </StyledRouterLink>
+      </LinkContainer>
+      <SearchBar />
+    </FlexContainer>
+    <ButtonContainer>
+      <StyledButton buttonText="Filters">SomeContent</StyledButton>
+      <StyledButton buttonText="Details">SomeContent</StyledButton>
```

Comment:
Should these be using LeafyGreen [Menu](https://www.mongodb.design/component/menu/example/) components?

Reply:
The component these trigger is a Card popup and the Menu component doesn't seem to be flexible enough to render it. It seems to only want to render a list of menu items.
<img width="612" alt="image" src="https://user-images.githubusercontent.com/4605522/186953469-8d78ec2d-6b49-4053-8ec3-b78585806370.png">